app/Http/Controllers/UsersController.php

```diff
@@ -12,8 +13,22 @@ class UsersController extends Controller
      */
     protected $northstar;
 
-    public function __construct(Northstar $northstar)
+    public function __construct(Registrar $registrar)
     {
-        $this->northstar = $northstar;
+        $this->registrar = $registrar;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $ids = User::all()->pluck('northstar_id')->all();
```

Comment:
I could see a need for that in the future. But this is only a page that displays users in the `Users` table which is really only admins, which is a small set of people right now. I mainly put this here to just have a visual display in the app of getting user information.